tests/CMakeLists.txt

```diff
@@ -72,3 +72,4 @@ ncnn_add_layer_test(UnaryOp)
 ncnn_add_layer_test(Mish)
 ncnn_add_layer_test(Swish)
 ncnn_add_layer_test(LSTM)
+ncnn_add_layer_test(yolov3detectionoutput)
```

Comment:
use the extract camelcase style class name here

Reply:
solved